<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongsWindow</class>
 <widget class="QMainWindow" name="SongsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MuzicUI - Songs</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/newPrefix/images/gradient.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="songsWindow_ExitBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	padding-top: 4px;
	border-image: none;
	color: white;
	background-color: #3F0071;
	border: none;
	border-radius: 5px;
}

QPushButton::hover {
	background-color: #9900F0;
}</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QFrame" name="songsWindow_SongsHolderFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>431</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
	background-color: ghostwhite;
	border-radius: 5px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="songsWindow_CareImgLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="assets/medias/qrc/care.qrc">:/newPrefix/images/mixaund-care.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_CareArtistLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mixaund</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_CareLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Care</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="songsWindow_CareListenBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-top: 4px;
	border-image: none;
	color: white;
	background-color: #3F0071;
	border: none;
	border-radius: 5px;
}

QPushButton::hover {
	background-color: #9900F0;
}</string>
     </property>
     <property name="text">
      <string>LISTEN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="songsWindow_HoldMeListenBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-top: 4px;
	border-image: none;
	color: white;
	background-color: #3F0071;
	border: none;
	border-radius: 5px;
}

QPushButton::hover {
	background-color: #9900F0;
}</string>
     </property>
     <property name="text">
      <string>LISTEN</string>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_HoldMeImgLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="assets/medias/qrc/holdme.qrc">:/newPrefix/images/purrple-cat-please-hold-me.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_HoldMeLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hold Me</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_HoldMeArtistLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Purple Cat</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_MorningLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>530</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Morning</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_MorningImgLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="assets/medias/qrc/morning.qrc">:/newPrefix/images/qlowdy-morning.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_MorningArtistLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Qlowdy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="songsWindow_MorningListenBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>570</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-top: 4px;
	border-image: none;
	color: white;
	background-color: #3F0071;
	border: none;
	border-radius: 5px;
}

QPushButton::hover {
	background-color: #9900F0;
}</string>
     </property>
     <property name="text">
      <string>LISTEN</string>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_ColorfulLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>530</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Colorful</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_ColorfulImgLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>350</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="assets/medias/qrc/colorful.qrc">:/newPrefix/images/roa-music-colorful.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="songsWindow_ColorfulArtistLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>500</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Roa-Music</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="songsWindow_ColorfulListenBtn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>570</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-top: 4px;
	border-image: none;
	color: white;
	background-color: #3F0071;
	border: none;
	border-radius: 5px;
}

QPushButton::hover {
	background-color: #9900F0;
}</string>
     </property>
     <property name="text">
      <string>LISTEN</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/medias/qrc/gradient.qrc"/>
  <include location="assets/medias/qrc/colorful.qrc"/>
  <include location="assets/medias/qrc/morning.qrc"/>
  <include location="assets/medias/qrc/holdme.qrc"/>
  <include location="assets/medias/qrc/care.qrc"/>
  <include location="assets/medias/qrc/music.qrc"/>
 </resources>
 <connections/>
</ui>
